docs/distros/ubuntu.md

```diff
@@ -27,7 +27,7 @@ sudo add-apt-repository ppa:serokell/tezos-rc && sudo apt-get update
 ## `tezos-baking` package
```

Comment:
A few lines above:
```
sudo apt-get install tezos-baker-011-pthangz2
```
is also worth updating, same for a couple of commands in the fedora doc.